src/helpers.ts

```diff
@@ -322,7 +322,7 @@ export function testSyntaxError(text: string, message?: string) {
   try {
     parse(text);
     throw new Error("SyntaxError not found");
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
I don't understand, shouldn't TypeScript narrow down type to `YAML.YAMLSyntaxError | YAML.YAMLSemanticError` after `isYAMLError` check? Then it should be an error, because `.source` is not a string.

Reply:
You are right, but for some reason the error type information is missing...

Reply:
You are right, but for some reason the error type information is missing...

Reply:
So, should we keep it as it is for now?

Yesterday, I tried


```ts
(error as  YAML.YAMLSyntaxError | YAML.YAMLSemanticError).message as unknown as string
```

But seems too stupid..

Reply:
> So, should we keep it as this way for now?

Yes, this part isn't import to users because it is a test helper. I'll look into this later...